stock_inventory_cost_info/models/stock_inventory.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class InventoryLine(models.Model):
+    _inherit = "stock.inventory.line"
+
+    currency_id = fields.Many2one(
+        string="Currency",
+        related="inventory_id.company_id.currency_id",
+        readonly=True,
```

Comment:
For related fields the default value is True, so you can delete this attribute

Reply:
As it works the same, please let me this time to not change it for not having to push + CIs.